index.bs

```diff
@@ -53,6 +61,73 @@ dictionary StorageBucketOptions {
 
 <h3 id="storage-bucket-open">Creating a bucket</h3>
 
+<div algorithm>
+
+  The <dfn method for="StorageBucketManager">open(|name|, |options|)</dfn> method steps are:
+
+  1. Let |environment| be [=/this=]'s [=/relevant settings object=].
+
+  1. Let |storageKey| be the result of running [=obtain a storage key=] given |environment|.
+
+      If failure is returned, then [=exception/throw=] a "{{SecurityError}}" {{DOMException}} and abort these steps.
+
+  1. Let |p| be [=a new promise=].
+
+  1. Run the following steps [=in parallel=]:
+
+      1. Let |r| be the result of running [=open a bucket=] with
+          |storageKey|,
+          |name|,
+          and |options| if given and undefined otherwise.
```

Comment:
The output doesn't format this with the newlines, so `|options| if given and undefined otherwise` doesn't scan as  a single clause to me. Instead, it reads like "Let |r| be [...] if given and undefined otherwise` which confused me.

(This is less confusing if `|options|` is not the last parameter.)

From looking at the IDB spec, it seems the way to handle this is to drop `and undefined otherwise` and to define the params for `open a bucket` as `and optional |options|`. I am basing this on [this part](https://www.w3.org/TR/IndexedDB/#ref-for-retrieve-multiple-values-from-an-object-store) of that spec.